tests/data/output/schema/version_in_subschema/output-json-schema.json

```diff
@@ -1,18 +1,19 @@
 {
-    "$$version": 1,
```

Comment:
I think this file was picked up by git as rename instead of new file and hence the diff is weird ?

Reply:
Yeah, probably.